resources/assets/components/utilities/Select.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { map } from 'lodash';
+
+const Select = ({ values, value, onChange }) => (
+  <div className="select">
+    <select value={value} onChange={event => onChange(event.target.value)}>
```

Comment:
Does the `value` attribute set the default value on `select`? (I thought that was only through the `options`s `selected` attribute?)

Reply:
It [does](https://reactjs.org/docs/forms.html#the-select-tag)! It actually yells at you if you use `selected`, which is what I'd originally done.

Reply:
Aha! Ya learn something new... 👨‍🏫